sjb/config/test_cases/azure_build_node_image_rhel_310.yml

```diff
@@ -10,17 +10,23 @@ sync:
 parameters:
 - name: "AZURE_VM_SIZE"
   default_value: "Standard_D2s_v3"
+- name: "OPENSHIFT_RELEASE"
+  default_value: "3.10"
+- name: "OPENSHIFT_RELEASE_LATEST"
```

Comment:
I am not sure we need this parameter. Playbooks are already versioned based on the branch (we know that playbooks run from release branches are not latest).

Reply:
as discusses on irc will move to maintain this in the ansible code via release branches.